contrib/test/integration/results.yml

```diff
@@ -0,0 +1,62 @@
+---
+# vim-syntax: ansible
+
+- hosts: '{{ hosts | default("all") }}'
+  vars_files:
+    - "{{ playbook_dir }}/vars.yml"
+  vars:
+    _result_filepaths: []  # do not use
+    _dstfnbuff: []  # do not use
+  tasks:
+    - name: The crio_integration_filepath is required
+      tags:
+          - integration
+      set_fact:
+       _result_filepaths: "{{ _result_filepaths + [crio_integration_filepath] }}"
+
+    - name: The crio_node_e2e_filepath is required
```

Comment:
In any given job we only have e2e _or_ integration results, right?

Reply:
It depends on what tags are used (if any).  If all tags are active, then both tests run and we expect both results.  If some combination of tags are used, only those results are collected.  I s'pose the assumption is the same tagging is used for the playbook and result-collection.  That could be a bit problematic.

I'm a little loath to combine the two (testing + result collection), but that's an option I'd consider.  It would mean possibly living with having no results, if/when the playbook itself fails (possibly due to testing actions).  Perhaps that's not an issue if there's some tertiary artifact/logging going on.

Reply:
Ah, sorry - I glossed over the tags here. This sounds good. I like the tags controlling everything. I agree we can run this as a post-job step to ensure it happens even if any of the build steps fail.